generators/uscore-r4/templates/module.yml.erb

```diff
@@ -23,3 +23,4 @@ test_sets:
         sequences:<% sequences.each do |sequence| %>
         - <%=sequence[:classname]%><% end %>
         - R4ProvenanceSequence
```

Comment:
As an aside for a future PR we should consider if Provenance tests belong in the US Core module.  I didn't see any Provenance requirements in the [IG](https://www.hl7.org/fhir/us/core/index.html)

Reply:
Provenance isn't there yet, but it will be: https://confluence.hl7.org/display/SEC/Basic+Provenance+Implementation+Guide

Reply:
Here is the latest CI version of US Core R4:
http://build.fhir.org/ig/argonautproject/R4/branches/master/basic-provenance.html